AndroidBroadcastReceiverAssignment/app/src/main/java/com/weboniselab/meghana/android/assignment6/androidbroadcastreceiverassignment/BroadcastHomePage.java

```diff
@@ -0,0 +1,53 @@
+package com.weboniselab.meghana.android.assignment6.androidbroadcastreceiverassignment;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.telephony.SmsManager;
+import android.telephony.SmsMessage;
+import android.telephony.TelephonyManager;
+import android.util.Log;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.widget.Toast;
+
+public class BroadcastHomePage extends BroadcastReceiver {
```

Comment:
Not Good BroadcastReeiver Name.

It should be MessageReceiver or something like that.